test/parallel/test-readline-interface.js

```diff
@@ -613,6 +613,43 @@ function isWarned(emitter) {
       rli.close();
     }
 
+    {
+      // Back and Forward one character
+      const fi = new FakeInput();
+      const rli = new readline.Interface({
+        input: fi,
+        output: fi,
+        prompt: '',
+        terminal: terminal
+      });
+      fi.emit('data', 'the quick brown fox');
+      let cursorPos = rli._getCursorPos();
+      assert.strictEqual(cursorPos.rows, 0);
+      assert.strictEqual(cursorPos.cols, 19);
+
+      // back one character
```

Comment:
tiniest of nits: Can you use caps for the first letter in each comment.

Reply:
Fixed them. Thanks 😄